src/gui_common/tooltip/ToolTipManager.tscn

```diff
@@ -530,6 +530,79 @@ margin_bottom = 204.0
 [node name="ModifierList" parent="GroupHolder/organelleSelection/chemoSynthesizingProteins/MarginContainer/VBoxContainer" index="3"]
 margin_bottom = 138.0
 
+[node name="storage" type="HBoxContainer" parent="GroupHolder/organelleSelection/chemoSynthesizingProteins/MarginContainer/VBoxContainer/ModifierList" index="0"]
+margin_right = 318.0
+margin_bottom = 20.0
+mouse_filter = 2
+script = ExtResource( 21 )
+DisplayName = "STORAGE"
+ModifierValue = "+0.5"
+ModifierValueColor = Color( 0, 1, 0, 1 )
+ModifierIcon = ExtResource( 12 )
+
+[node name="Value" type="Label" parent="GroupHolder/organelleSelection/chemoSynthesizingProteins/MarginContainer/VBoxContainer/ModifierList/storage"]
+margin_top = 1.0
+margin_right = 16.0
+margin_bottom = 18.0
+custom_fonts/font = ExtResource( 26 )
+custom_colors/font_color = Color( 0, 1, 0, 1 )
+text = "+0.5"
+
+[node name="Name" type="Label" parent="GroupHolder/organelleSelection/chemoSynthesizingProteins/MarginContainer/VBoxContainer/ModifierList/storage"]
+margin_left = 20.0
+margin_right = 73.0
+margin_bottom = 19.0
+custom_fonts/font = ExtResource( 27 )
+text = "Storage"
+__meta__ = {
+"_editor_description_": "PLACEHOLDER"
+}
```

Comment:
Shouldn't this string be translatable? Are the other organelles like this as well, @Kasterisk ?

Reply:
This is just a flavor text for display only in the editor, the actual string will be set by code which is already translatable. The other organelle tooltips are also like this (if I'm not misremembering my own design).

Reply:
Ah, okay, I guess I don't seem to know enough about this. I was also a bit confused as the numbers where in two different places, but I guess that's also the editor only stuff?
So that just leaves one of the storage change numbers not being also put in a tooltip as a problem.

Reply:
Yep, the organelle tooltip system warrants an overhaul eventually.